src/Workspaces/VisualBasic/Portable/CodeCleanup/Providers/ReduceTokensCodeCleanupProvider.vb

```diff
@@ -221,6 +253,19 @@ Namespace Microsoft.CodeAnalysis.CodeCleanup.Providers
                 Return valueText
             End Function
 
+            Private Function GetIntegerLiteralValueString(value As Long, base As LiteralBase) As String
+                Select Case base
+                    Case LiteralBase.Decimal
+                        Return value.ToString()
```

Comment:
Be careful with culture here -- you may wish to explicitly pass CultureInfo.InvariantCulture in case this is running on some platform that tries putting commas as separators.